### PR TITLE
Working mode proposal

### PR DESCRIPTION
**Pull request by cramforce**

### CODE REVIEW

**Comment by cramforce** [comment]
CC @tobie @mrjoro

**Review by paularmstrong — APPROVED** [review]

File: WORKING_MODE.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Working mode of the AMP TSC
+
+- We meet every 2 weeks for 1h via video conference.
+- We meet in person 2x per year (timed around the major AMP conference "AMP Conf" and "AMP Contributor Summit").
+- Meeting notes are published in this repository.
+- We [create an issue](https://github.com/ampproject/meta-tsc/labels/TSC%20Meeting) for every meeting to document agenda and notes.
+- Members of the Advisory Committee, Fascilitators of Working Groups, and Members of the Approvers Working Group may add agenda items via comments on the respective issue at any time.
```
Comment (rudygalfi) [inline WORKING_MODE.md]: spelling: Facilitators

**Comment by tobie** [comment]
For reference, [here](https://docs.google.com/document/d/10mzKsZ9zRllxkgRWNA5q2SDmj7P6Lajf2YPlJSwwj60/edit)'s the draft charter for the AC.

**Comment by tobie** [comment]
I'm curious why the Approvers WG is singled out in the work mode document. In what is that group different from the other WGs?

Maybe we should have a page where we list all the WGs and what each one of them is responsible for.

**Comment by mrjoro** [comment]
LGTM!

@tobie there's a [pending PR to document the Working Groups](https://github.com/ampproject/meta/pull/13).

**Comment by tobie** [comment]
Awesome. So I feel like the whole section about the Approvers WG should either be made generic (i.e. about all WGs) or be removed.

**Comment by davidstrauss** [comment]
> Awesome. So I feel like the whole section about the Approvers WG should either be made generic (i.e. about all WGs) or be removed.

I agree.

**Comment by sveronesi** [comment]
Perfect

File: WORKING_MODE.md (hunk 1 above)
Comment (cramforce) [reply WORKING_MODE.md]: Fixed

**Comment by cramforce** [comment]
I definitely think the Approvers group is special in that it introduced a layer of indirection in the decision making process that isn't required by the governance doc; and the Approvers group is different from all other working group in that it's only job is decision making of work owned by another working group.

**Comment by davidstrauss** [comment]
I was partly agreeing with @tobie for the reasons he stated and partly because I've seen even approval working groups see significant refactoring even on mature projects (e.g. Fedora), making me reluctant to memorialize them more than necessary.

However, I hold this opinion very weakly. Please don't hold off on merging on my account.

File: WORKING_MODE.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Working mode of the AMP TSC
+
+- We meet every 2 weeks for 1h via video conference.
```
Comment (cramforce) [inline WORKING_MODE.md]: Call out specifically that we see this as status quo and something we'd adapt as we see fit

**Review by cpapazian — APPROVED** [review]

**Review by dvoytenko — APPROVED** [review]

File: WORKING_MODE.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+- We meet in person 2x per year (timed around the major AMP conferences "AMP Conf" (Spring) and "AMP Contributor Summit" (Fall)).
+- Meeting notes are published in [this repository]((https://github.com/ampproject/meta-tsc/labels/TSC%20Meeting)).
```
Comment (dvoytenko) [inline WORKING_MODE.md]: nit: link format is broken a bit with double parens?

**Review by dvoytenko — APPROVED** [review]

**Review by davidstrauss — APPROVED** [review]

File: WORKING_MODE.md (hunk 3 above)
Comment (tobie) [inline WORKING_MODE.md]:
```suggestion
- Meeting notes are published in [this repository](https://github.com/ampproject/meta-tsc/labels/TSC%20Meeting).
```

Comment (cramforce) [reply WORKING_MODE.md]: Done

**Pull request merged**